04 Event Emitters/assets/js/app/components/TableComponent.js

```diff
@@ -0,0 +1,59 @@
+(function ComponentInitializator($, React, App) {
+  'use strict';
+
+  var createCell = function (type, text) {
+    return React.createElement(type, null, text);
+  };
+
+  var createRow = function (cell, index, contact) {
+    return React.createElement('tr', { key: index },
+      createCell(cell, contact.name),
+      createCell(cell, contact.phone),
+      createCell(cell, contact.email)
+    );
+  };
+
+  var createThead = function () {
+    var contactTemplate = {
```

Comment:
I did refactor `createRow` and `createCell` (before called `createTheadRow`, `createTbodyRow` and `createCell`) after I wrote `createThead` and `createTbody`. Should I rename `createTbody` to `createBody` ?